### PR TITLE
Feature: Added tooltip to show message sender name.

### LINKED ISSUE #232: Feature: show sender name in message tooltip

### PR DESCRIPTION
**Pull request by iamprayush**
Fixes #232

- Used [ui-kitten's tooltip](https://akveo.github.io/react-native-ui-kitten/docs/components/tooltip/overview/#tooltip).
- Upon long-pressing a message bubble, the tooltip will show the sender's name, just like in the web app.

tooltip_demo

### CODE REVIEW

**Comment by muhsin-k** [comment]
@iamprayush Thanks for the PR ❤️. Currently, we are using [UI Kitten](https://akveo.github.io/react-native-ui-kitten/) as the main UI framework. We suggest using [tooltip](https://akveo.github.io/react-native-ui-kitten/docs/components/tooltip/overview#tooltip) provided by UI kitten. Adding additional package will increase bundle size.

**Comment by iamprayush** [comment]
> @iamprayush Thanks for the PR . Currently, we are using [UI Kitten](https://akveo.github.io/react-native-ui-kitten/) as the main UI framework. We suggest using [tooltip](https://akveo.github.io/react-native-ui-kitten/docs/components/tooltip/overview#tooltip) provided by UI kitten. Adding additional package will increase bundle size.

@muhsin-k I've now removed react-native-elements and used the ui-kitten tooltip instead. I've also updated the demo gif, PTAL!

**Comment by muhsin-k** [comment]
@iamprayush We will test it out in both Android and iOS and let you know if any change is required

File: src/components/ChatMessageItem.js [hunk 1]
```diff
@@ -163,6 +175,16 @@ const ChatMessageItemComponent = ({ type, message, eva: { style, theme }, create
...
     </TouchableOpacity>
   );
+
+  return (
+    <Tooltip
+      anchor={renderChatMessageIconComponent}
+      visible={tooltipVisible}
+      onBackdropPress={() => setTooltipVisible(false)}
+      placement="top">
+      <Text style={style.tooltipText}>Sent by: {message.sender.name}</Text>
```
Comment (muhsin-k) [inline src/components/ChatMessageItem.js:185]: @iamprayush The app will crash if the message sender is empty (Some messages doesn't have sender details). So show tooltip only for messages which contain sender details.

Comment (iamprayush) [reply src/components/ChatMessageItem.js]: I've added the tooltip conditionally now, PTAL!

**Review by muhsin-k — APPROVED** [review]
Looks good to me

**Pull request merged**